DQMOffline/Trigger/python/BTagAndProbeMonitoring_cff.py

```diff
@@ -0,0 +1,129 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.BTagAndProbeMonitor_cfi import BTagAndProbeMonitoring
+
+from Configuration.Eras.Modifier_run2_HLTconditions_2018_cff import run2_HLTconditions_2018
+from Configuration.Eras.Modifier_run2_HLTconditions_2017_cff import run2_HLTconditions_2017
+from Configuration.Eras.Modifier_run2_HLTconditions_2016_cff import run2_HLTconditions_2016
+
+###
+### Ele+Jet
+###
+
+BTagAndProbe_1e1m = BTagAndProbeMonitoring.clone()
```

Comment:
```suggestion
BTagAndProbe_1e1m = BTagAndProbeMonitoring.clone(
...
)
```
(same as above, and please also remove the type specification for the cloned parameters)